mindjunkies/courses/management/commands/populate_course.py

```
@@ -0,0 +1,129 @@
+                Module.objects.create(
+                    course=course,
+                    title=f"Module {i+1}: {fake.sentence(nb_words=4)}",
+                    order=i
+                )
+
+            # Create enrollments and ratings
+            if students.exists():
+                num_enrollments = random.randint(1, min(8, students.count()))
+                enrolled_students = random.sample(list(students), num_enrollments)
+                for student in enrolled_students:
+                    enrollment = Enrollment.objects.create(
```

Ruff (0.8.2)

Local variable `enrollment` is assigned to but never used

Remove assignment to unused variable `enrollment`

(F841)